draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -833,16 +835,14 @@ If any step fails, the SD-JWT is not valid, and processing MUST be aborted. Othe
 
 Note that these processing steps do not yield any guarantees to the Holder about having received a complete set of Disclosures. That is, for some digest values in the Issuer-signed JWT (which are not decoy digests) there may be no corresponding Disclosures, for example, if the message from the Issuer was truncated.
 
```

Comment:
```suggestion
It is up to the Holder how to maintain the mapping between the Disclosures and the plaintext claim values to be able to display them to the user when needed.
```

Reply:
or perhaps
```suggestion
It is up to the Holder's software how to maintain the mapping between the Disclosures and the plaintext claim values to be able to display them to the user when needed.
```

Reply:
Everything here is about the user, so I would go for the first version. I initially removed the sentence because it didn't add much value and deepens the potential Holder/user confusion. But I'm fine with keeping it or a suitable replacement.

Reply:
I kinda presumed as much. I do think it adds some value so would like something suitable kept or replaced.

Reply:
+1 to the first proposal of the addition